Assets/PatchKit Patcher/Scripts/Debug/UnityMessageWriter.cs

```diff
@@ -19,6 +19,11 @@ public UnityMessageWriter([NotNull] IMessageFormatter formatter)
         public void Write(Message message, MessageContext messageContext)
         {
             var text = _formatter.Format(message, messageContext);
+            var patcher = Patcher.Instance;
+            if (!String.IsNullOrEmpty(patcher.AppSecret))
```

Comment:
C# guidelines state that uppercase versions of primitive shouldn't be used. Should be `string` instead of `String`.

Reply:
Done